package.json

```diff
@@ -9,7 +9,7 @@
     "migrate": "knex migrate:latest --knexfile config/knexfile.js",
     "rollback": "knex migrate:rollback --knexfile config/knexfile.js",
     "watch:dev": "nodemon",
-    "server": "supervisor server.js",
+    "watch:dev:windows": "supervisor server.js",
```

Comment:
This script looks much more readable and other developers who work on Windows machines now easily can run the server